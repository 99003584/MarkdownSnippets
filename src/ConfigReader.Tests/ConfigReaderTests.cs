using System.IO;
using System.Threading.Tasks;
using VerifyXunit;
using Xunit;
using Xunit.Abstractions;

public class ConfigReaderTests :
    VerifyBase
{
    [Fact]
    public Task Empty()
    {
        var config = ConfigReader.Parse("{}");

        return Verify(config);
    }

    [Fact]
    public Task Values()
    {
        var stream = File.ReadAllText("sampleConfig.json");
        var config = ConfigReader.Parse(stream);
        return Verify(config);
    }

    public ConfigReaderTests(ITestOutputHelper output) :
        base(output)
    {
    }
}