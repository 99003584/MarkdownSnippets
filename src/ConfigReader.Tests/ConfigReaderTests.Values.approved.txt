{
  WriteHeader: true,
  ReadOnly: false,
  LinkFormat: 'Tfs',
  TocLevel: 3,
  UrlsAsSnippets: [
    'Url1',
    'Url2'
  ],
  Exclude: [
    'Dir1',
    'Dir2'
  ]
}